.circleci/config.yml

```diff
@@ -166,14 +166,25 @@ jobs:
           condition: <<parameters.use_review_app>>
           steps:
             - run:
-                name: Set Heroku Review Apps URL for Pull Requests
+                name: Run E2E tests (same instance)
                 command: |
-                  echo "export E2E_BASE_URL_REVIEW_APP=$(sh .circleci/get-review-app-url.sh)" >> $BASH_ENV
+                  echo "export REDIS_URL=''" >> $BASH_ENV
+                  echo "export AUTH_PROVIDER_URL='http://localhost:3999'" >> $BASH_ENV
+                  echo "export NOMIS_ELITE2_API_URL='http://localhost:3999'" >> $BASH_ENV
+                  echo "export SERVER_HOST='localhost:3000'" >> $BASH_ENV
+                  echo "export E2E_BASE_URL_REVIEW_APP='http://localhost:3000'" >> $BASH_ENV
                   source $BASH_ENV
-                no_output_timeout: 600s
-      - run:
-          name: Run E2E tests
-          command: E2E_BASE_URL=$<<parameters.base_url_env>> npm run test-e2e:ci
+                  npm run build
```

Comment:
Why not just run this as a separate `run` command? 

Might make it easier to read both here and in the CI steps. So something like:

```yaml
- run:
    name: Build app
    command: npm run build
- run:
    name: Start the app
    command: 
        node start.js 1>/dev/null &
        node mocks/auth-server.js &
        sleep 5
- run:
    name: Run E2E tests
    command: E2E_BASE_URL=$<<parameters.base_url_env>> npm run test-e2e:ci
```

Reply:
I tried that.

The processes get terminated. So they have to kick off in the same container process(?) as the actual e2e tests

Reply:
Ahhh gotcha! Makes sense

Reply:
Could probably still split the build though?